### ISSUE TITLE
helm: CSV generation doesn't populate displayName or description fields for CRDs

### ISSUE THREAD

**Issue by jsm84** [operator-framework/operator-sdk#2581]
## Bug Report

**What did you do?**
Create a CSV using the following command:
`operator-sdk olm-catalog gen-csv ...`

**What did you expect to see?**
The following fields should exist in the CSV for each CRD that is owned by the operator:
spec.customresourcedescriptions.owned[].description
spec.customresourcedescriptions.owned[].displayName

**What did you see instead? Under which circumstances?**
The above fields are missing from the generated CSV. This causes OLM deployment to fail, as these fields are required.

**Environment**
* operator-sdk version: 

"v0.15.0-19-g5709ff37", commit: "5709ff37ebb293d4a1f5990f4e5bbf23ce26be6a", go version: "go1.13.6 linux/amd64"

* go version:

go version go1.13.6 linux/amd64

* Kubernetes version information:

Client Version: openshift-clients-4.2.2-201910250432-4-g4ac90784
Server Version: 4.2.16
Kubernetes Version: v1.14.6+8bbaf43

* Kubernetes cluster kind: 

OpenShift

* Are you writing your operator in ansible, helm, or go?

This affects all operator development methods.

**Possible Solution**

**Additional context**

**Comment by andrewazores** [comment]
Is this the same as #2575?

**Comment by estroz** [comment]
@andrewazores yes this is a similar misunderstanding as in #2575 (see my closing comment in that issue for `displayName` issues).

As for `description`, that should always be set from comments above your kind's type. You're saying it is not?

**Comment by jsm84** [comment]
Yes, this would be somewhat of a duplicate (however with the addition of the description field). In this case, I'm using a helm operator. This was discovered yesterday in a dry-run of a partner workshop, which will be used for start-to-finish onboarding of partners who have a certified operator and also a helm chart, who would like to create and certify an operator (that's neither here nor there, just provided for context, if not priority).

The partner experience for creating a helm operator is a bit bumpy, to say the least. In this case I would expect that running the csv-gen tool would yield a working CSV that would deploy in OLM, without having to add missing fields manually. In cases where a user/partner won't have a golang operator with a struct/type defined with comments, could this instead default to "Placeholder Name" (or even better the CR kind) and "Placeholder CR Description"?

I realize this may be more of a feature request than a bug, but the "bug" IMO is the fact that the csv-gen tool doesn't yield a working CSV without having to add these missing fields.

**Comment by estroz** [comment]
@jsm84 ah I see. The CSV generator does not officially support Helm (or Ansible) operators yet so usage with non-Go behavior is technically unknown.

Currently _some_ required fields are reported as missing when running the tool. What we need to do is use this [api validation library](https://github.com/operator-framework/api) to report which fields need populating. Then the user will at least be aware that their CSV is invalid. This is a priority, and I'll mark this as such.

**Comment by camilamacedo86** [comment]
With the SDK 1.0 release will be possible to integrate Ansible/Helm with OLM via the makefile targets `make bundle` and ` bundle-build`. Also, users are indeed able to generate the package manifests as well by adding the following target in their Makefile :  

```
# Options for "packagemanifests".
ifneq ($(origin CHANNEL), undefined)
PKG_CHANNELS := --channel=$(CHANNEL)
endif
ifeq ($(IS_CHANNEL_DEFAULT), 1)
PKG_IS_DEFAULT_CHANNEL := --default-channel
endif
PKG_MAN_OPTS ?= $(PKG_CHANNELS) $(PKG_IS_DEFAULT_CHANNEL)

# Generate package manifests.
packagemanifests: kustomize
	operator-sdk generate kustomize manifests -q --interactive=false
	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
	$(KUSTOMIZE) build config/manifests | operator-sdk generate packagemanifests -q --version $(VERSION) $(PKG_MAN_OPTS) 
```

In this way, I am closing this one as solved for 1.0. 
c/c @estroz
